### ISSUE TITLE
deploy contracts not working on Ubuntu 18

### ISSUE THREAD

**Issue by j4ys0n** [ethereum-oasis/baseline#104]
## Expected Behavior
`docker-compose run --rm deploy sh deploy.sh` should complete successfully.

## Actual Behavior
Error

## Steps to Reproduce the Problem
note: Some build steps had to be updated because pushd/popd wasn't working in npm scripts on Ubuntu.

  1. build steps for Ubuntu are here: https://github.com/splunk/baseline/tree/integrate-splunk#ubuntu

## Specifications

  - Version:
  - Platform: Ubuntu 18, Node.js v11, Docker 19.03.8
  - Subsystem:

```
> radish-34@1.0.0 deploy /home/jayson/Code/baseline/radish34
> docker-compose run --rm deploy sh deploy.sh

WARNING: Found orphan containers (splunk, cadvisor) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
Creating redis-supplier2    ... done
Creating redis-supplier1    ... done
Creating geth-bootnode      ... done
Starting mongo-buyer     ... done
Creating mongo-supplier1    ... done
Creating mongo-supplier2    ... done
Creating radish34_ganache_1 ... done
Starting zkp                ... done
Creating redis-buyer        ... done
Creating geth-miner1        ... done
Creating geth-miner2        ... done
Creating geth-node          ... done
Creating messenger-buyer     ... done
Creating messenger-supplier1 ... done
Creating messenger-supplier2 ... done
Patiently waiting 10 seconds for ganache container to init ...
Checking for ganache ...
✅  ERC1820Registry deployed: 0x448de9B34ac4DD0901DCc3f2fF1a31822B51a397
✅  OrgRegistry deployed: 0x31088fd0eede771d5bda1558e06a666Cd9BF110c
✅  BN256G2 library deployed: 0x8f17969A8dc9cbAe2EB98541F33c7c396f615241
✅  Verifier deployed: 0xDf3C747B74CeFe4ffEa5baa2D0eAFE2B0F86A8F3
✅  Shield deployed: 0x7370f1C710F3af6f28Be19ed99e0ed8f1B59b1CB
(node:8) UnhandledPromiseRejectionWarning: Error: incorrect number of arguments
    at Contract.assignManager (/app/node_modules/ethers/contract.js:130:19)
    at Object.assignManager (/app/src/utils/organization.js:29:39)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async assignManager (/app/src/deploy.js:39:31)
    at async main (/app/src/deploy.js:115:3)
(node:8) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:8) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by j4ys0n** [comment]
link to the error in `ethers` https://github.com/ethers-io/ethers.js/blob/master/contract.js#L130

**Comment by skarred14** [comment]
the above error was resolved in #95, is this still an issue?

**Comment by j4ys0n** [comment]
That fixed it! (I just pulled in the latest from master into my fork)

Thank you!
